packages/react/src/primitives/types/fieldGroup.ts

```diff
@@ -1,7 +1,10 @@
 import { FlexProps } from './flex';
 
+type FieldGroupOrientation = 'horizontal' | 'vertical';
```

Comment:
Types used in other exported types must be exported

```suggestion
export type FieldGroupOrientation = 'horizontal' | 'vertical';
```